starboard/shared/win32/posix_emu/include/sys/socket.h

```diff
@@ -25,6 +26,25 @@ extern "C" {
 int sb_socket(int domain, int type, int protocol);
 #define socket sb_socket
 
+int sb_bind(int socket, const struct sockaddr* address, socklen_t address_len);
```

Comment:
Are `sockaddr` and `socket_t` ABI stable? Do we know the size and memory layout?

Reply:
sockaddr and sockaddr_in are idential in memory layout.
And socklen_t is an integer type of width of at least 32 bits.